app/assets/scripts/map.js

```diff
@@ -157,6 +157,40 @@ $(document).ready(function() {
     });
 });
 
+function getNextDate(startDate) {
+    date = startDate.getDate();
+    month = startDate.getMonth();
+    year = startDate.getFullYear();
+    MAX_MONTH = 11;
```

Comment:
Why 0-11 and not 1-12? I think 1-12 would make more sense. In any case, should add a docstring here imo.